src/request/sign-transaction/BaseLayout.js

```diff
@@ -97,6 +90,9 @@ class BaseLayout {
                 this._onConfirm(request, resolve, reject, passphrase);
             },
         );
+
+        // This cannot be 'CANCEL', because for checkout we need to go back to
+        // the CheckoutOverview in the Accounts Manager
         this._passphraseBox.on(PassphraseBox.Events.CANCEL, () => window.history.back());
```

Comment:
What can not be CANCEL? The Event name? I think the name is just fine, because the user cancels his entering of the passphrase.